Gopkg.toml

```diff
@@ -83,7 +83,7 @@
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "4.0.0"
+  branch = "release-4.0"
```

Comment:
Do you know what the difference is between `4.0.0` and `release-4.0`?

Reply:
I changed it when tried to use `dep ensure` in planet repo. But I still cannot fix dependency hell, caused by renaming Sirupsen -> sirupsen. After all fixes build of planet crashes on runc library.